### ISSUE TITLE
Bundles disappear

### ISSUE THREAD

**Issue by awreccan** [teresa-ou/inboxy#126]
Updates tab -> click into an email -> Click the "Inbox" link on the left nav bar -> Gmail will navigate you to the Primary tab -> Primary tab suddenly doesn't have bundles.

To resolve manually:
- refresh Gmail, OR
- Click into an email -> Press `u` to go back to inbox

**Comment by tmenier** [comment]
Glad to hear it's not just me.

**Comment by whitespacecode** [comment]
I went back to this build. It also includes bundles on colors and features like select all/ delete all
https://github.com/lpakula/inboxy/releases/tag/v0.0.10

**Comment by montagros** [comment]
I also have this problem

> I went back to this build. It also includes bundles on colors and features like select all/ delete all
> https://github.com/lpakula/inboxy/releases/tag/v0.0.10

In this build the problem also occurs.

**Comment by whitespacecode** [comment]
Did you remove the old inboxy and all ? Strange because that one works really well for me

**Comment by montagros** [comment]
I just turned it off in the extensions menu. I'll try to completely remove it. Thanks

**Comment by glenviewjeff** [comment]
> I just turned it off in the extensions menu. I'll try to completely remove it. Thanks

@montagros did that work for you? If so I'll give it a shot.

**Comment by montagros** [comment]
@glenviewjeff No, that didn't work either. Due to many errors, I started using Simplify Gmail (Canary), which has this function and many other useful ones, but unfortunately it is paid.

**Comment by glenviewjeff** [comment]
@montagros thanks for letting me know and for the tip. $2/month is pretty steep, but we of course do use Gmail a lot. I will definitely check it out. If there were a mobile Inbox app replacement to pair with it, I'd be in.

**Comment by awreccan** [comment]
FYI this is still a problem for me, though I've started resolving the disappearing bundles with the following step (instead of "Click into an email -> Press u to go back to inbox"):

- Press ` (this cycles from Primary to Promotions tab) followed by Shift+` (this cycles from Promotions tab to Primary)

which brings the bundles back. I've been doing this so long that this is now an automated muscle-memory bugfix instead of the code bugfix we need 😄

**Comment by benoror** [comment]
Hi! Losing bundles when navigating Updates → thread → Inbox is exactly the kind of path we have been hardening. **Inbundly** soft-retries bundling after navigation (v2.1.1+) and can **remember the open bundle** across Inbox nav (v4.3.0, session).

Please try the latest build. If Primary still comes back unbundled until refresh, open a bug with the exact click path.

Chrome: https://chromewebstore.google.com/detail/inbundly/pbfjicjmcpogjlbpljebhhgkgfhbdcga
Issues: https://github.com/benoror/inbundly/issues/new
